public_activity.gemspec

```diff
@@ -31,7 +31,7 @@ Gem::Specification.new do |s|
   when 'active_record'
     s.add_dependency 'activerecord', '>= 3.0'
   when 'mongoid'
-    s.add_dependency 'mongoid',      '~> 3.0'
+    s.add_dependency 'mongoid',      '>= 3.0'
```

Comment:
Keep it at `~> 3.0` please, once a 4.0 comes it might break compatibility and I don't want that.